mail-manager/src/main/scala/it/pagopa/interop/commons/mail/MailData.scala

```diff
@@ -47,6 +47,14 @@ object Mail {
     def make(c: Context)(args: c.Expr[Any]*): c.Expr[InternetAddress] = apply(c)(args: _*)
   }
 
+  /**
+    *
+    * @param text comma separated emails list
+    * @return
+    */
```

Comment:
🧹

Reply:
I put it intentionally: without it's not clear that that text could be a comma-separated list of emails.

Reply:
I could agree about the style, but I'm afraid I have to disagree about the return type List means comma-separated input: anything in input it's saying that.

Anyway, an example could be useful. 👍